ansible/roles/products/hand-e/deploy/meta/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+dependencies:
+  - { role: docker/docker-image }
+  - { role: products/hand-e/docker-container }
+  - { role: products/hand-e/desktop-icons }
+  - { role: products/hand-e/cyberglove, when: use_cyberglove }
```

Comment:
use_cyberglove or use-cyberglove now?

Reply:
I am allowing variable names to still have underscore, but file names are forbidden from having underscore. So use_cyberglove is still valid because it is a (boolean) variable